src/cursor/cur_hs.c

```diff
@@ -616,9 +622,12 @@ __curhs_search_near_helper(WT_SESSION_IMPL *session, WT_CURSOR *cursor, bool bef
     } else {
         /*
          * If we want to land on a key that is larger or equal to the specified key, keep walking
-         * forwards as there may be content inserted concurrently.
+         * forwards as there may be content inserted concurrently. Note that there is an
```

Comment:
hey @bvpvamsikrishna , we have talked about this edge case where if we set the key with btree id and a key that records can appear only after the cursor position. But what about the case where we attempt to get the last positioned record of the btree id and key? Can't records be placed after the cursor position here and thus won't be deemed as the "last" entry? Does this case matter?